### ISSUE TITLE
Function 4 : Emailer

### ISSUE THREAD

**Issue by elabandari** [UBC-MDS/pyhousehunter#3]
Function 4 takes in a pandas dataframe or .csv file with from Function 3 and emails it to the end-user.

**Comment by elabandari** [comment]
Documenting some of the decisions made in designing and implementing the emailer function.

These points were discussed in the group meeting on Friday February 26th.

1) Should the user have the ability to request the results in different forms (e.g. a csv or pandas dataframe object)?  Ultimately, the group decided that reading a .csv file into a pandas dataframe object was easy enough that the .csv file would be sufficient.
2) Should the user send the email to themselves from their own email address or show there be a central pyhousehunter email address?  We decided against the user sending themselves the results because they would need a gmail account and they would need to authorize access from less secure apps to their gmail account. This put the user's email at potential risk. Instead we set up an gmail account for pyhousehunters and send emails from there. We also plan to set up automation on the pyhousehunters gmail account where emails are not stored in the sent folder after they are sent (to protect user privacy). 
3) Should the users have the ability to change the email subject and email message? This may or may not be a desired functionality so we are thinking of having this as an optional argument with a default message.
